static/styles/portico/markdown.css

```diff
@@ -310,6 +310,8 @@
     pre {
         line-height: 15px;
         code {
+            font-size: 16px;
+            font-family: monospace;
```

Comment:
It makes no sense for the `line-height` to be less than the `font-size`. And the current font stack here is `Monaco,Menlo,Consolas,"Courier New",monospace`; while replacing it with the generic `monospace` alias might look better on your system, it’s going to look worse on others.

Reply:
Thanks for the review @andersk ! Made the required changes.